spec/geocode_and_hike_api_service_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+RSpec.describe 'GeoCodeService & Hiking Project API Service' do
+
+  def app
+    GeoCodeService
+  end
+
+  it 'should get lat and lng from location GeoCode API' do
+    
+    service = app.new
+    location_info = service.location_info("denver, co")[:geometry][:location]
+    expect(location_info[:lat]).to be_a Float
+    expect(location_info[:lng]).to be_a Float
+  end
+  
+  it 'Hiking service API' do
+    service = app.new
+    location_info = service.location_info("denver, co")[:geometry][:location]
+     service = HikingProjectService.new(location_info)
```

Comment:
Using the lat and long from above to get hiking trail information from services.